admin/mail.php

```diff
@@ -64,10 +64,10 @@
 
             if ($mailer->send_to_group($_REQUEST['to'])) {
                 $title  = T_('E-mail Sent');
-                $body   = T_('Your E-mail was successfully sent.');
+                $body   = T_('Your e-mail was successfully sent.');
             } else {
                 $title     = T_('E-mail Not Sent');
```

Comment:
"sent"?

Reply:
this is returning on success so this part is sent. (rewriting this)